components/grid/selection/rows.md

```diff
@@ -0,0 +1,276 @@
+}
+else
+{
+    <div>You can <strong>click both the checkbox or the row</strong> to select or deselect the row.</div>
+}
+<br/>
+<TelerikCheckBox @bind-Value="@CheckBoxOnlySelection"
+                 Id="@CheckboxId"/>
+<label for="@CheckboxId">Toggle checkbox only selection</label>
+<br />
+<br />
+<TelerikGrid Data=@GridData
+             SelectionMode="@GridSelectionMode.Multiple"
+             Pageable="true">
+    <GridColumns>
+        <GridCheckboxColumn SelectAll="@ShouldSelectAll" CheckBoxOnlySelection="@CheckBoxOnlySelection"></GridCheckboxColumn>
+        <GridColumn Field=@nameof(Employee.Name) Title="Name" />
+        <GridColumn Field=@nameof(Employee.Team) Title="Team" />
+    </GridColumns>
+</TelerikGrid>
+
+@code {
+    private List<Employee> GridData { get; set; }
+
+    private bool ShouldSelectAll { get; set; } = true;
+    private bool CheckBoxOnlySelection { get; set; }
+
+    private string CheckboxId { get; set; } = "checkboxOnlySelection";
+
+    protected override void OnInitialized()
+    {
+        GridData = new List<Employee>();
+        for (int i = 0; i < 15; i++)
+        {
+            GridData.Add(new Employee()
+                {
+                    EmployeeId = i,
+                    Name = "Employee " + i.ToString(),
+                    Team = "Team " + i % 3
+                });
+        }
+    }
+
+    public class Employee
+    {
+        public int EmployeeId { get; set; }
+        public string Name { get; set; }
+        public string Team { get; set; }
+    }
+}
+````
+
+## Selected Rows
+
+* You can get or set the selected rows through the `SelectedItems` property. It is a collection of the [Grid's `Data`]({%slug grid-data-binding%}) model.
+* You can use the `SelectedItems` collection in two-way binding. You can predefine the selected item for your users through the two-way binding of the `SelectedItems` property. The collection will be updated by the Grid when the selection changes.
+
+### Selected Rows When Data Changes
+
+When the Grid `Data` collection changes, the `SelectedItems` collection has the following behavior:
+
+* When you update a selected item in the Grid, you have to make the same update in the `SelectedItems` collection through the Grid [editing events]({%slug components/grid/editing/overview%}).
+* When you delete a selected item in the Grid, it will automatically delete from the `SelectedItems` collection. If you are using one-way binding for the `SelectedItems` collection and the [`SelectedItemsChanged` event](#selecteditemschanged), when you delete a selected item, the event fires. When you delete all selected items, the `SelectedItemsChanged` event fires with an empty collection.
+* When you create an item in the Grid, and you want to select it at the same time, use the Grid [editing events]({%slug components/grid/editing/overview%}).
+
+### Selected Rows Equals Comparison
```

Comment:
This whole section is 99% the same in the Row and Cell pages. Consider moving it to the Overview page.

Reply:
This will change the idea of the Overview page:
- to contain only the most important information in a summary
- to redirect for more information to the row and cell articles
I prefer to leave it as it is, so when the user is reading the row/cell article, they will have the full information in one place